client/my-sites/stats/info-panel/index.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
I am disabling classname check for this file, for now. The stylesheet is huge and breaking them into multiple smaller sheets+importing them feels like something that one can handle in a separate PR. What do you think?